.gitlab-ci.yml

```diff
@@ -124,15 +124,16 @@ benchmarks:
   stage:                           benchmark
   <<:                              *docker-env
   <<:                              *collect-artifacts
-  <<:                              *schedule-refs
+  # <<:                              *schedule-refs
```

Comment:
remove?!

```suggestion
```

Reply:
it was temporary, I tested if job schedules on a particular runner